### DISCUSSION TITLE
200行代码实现基于paxos的kv存储 - OpenACID Blog

### DISCUSSION THREAD

**Discussion by giscus** [openacid/openacid.github.io#37] [General]
# 200行代码实现基于paxos的kv存储 - OpenACID Blog

用200行代码实现一个基于paxos的kv存储, 以最简洁的形式展示paxos如何运行, 作为 paxos的直观解释 这篇教程中的代码示例部分

https://blog.openacid.com/algo/paxoskv/

**Comment by brucezo** [comment]
哈哈，大佬，爱了爱了。

**Comment by tong-hao** [comment]
Challenge failed, over 200 lines.

**Reply by drmingdrmer** [reply]
You are totally right!

I'm gonna have to update the title to `206 行代码实现...`

<img width="694" height="167" alt="image" src="https://github.com/user-attachments/assets/251632fa-3985-4d53-9ae8-3a78c371657c" />
